arch/arm/soc/nordic_nrf/nrf52/dts.fixup

```diff
@@ -64,6 +64,11 @@
 #define CONFIG_WDT_NRF_IRQ		NORDIC_NRF_WATCHDOG_40010000_IRQ_WDT
 #define CONFIG_WDT_NRF_IRQ_PRI		NORDIC_NRF_WATCHDOG_40010000_IRQ_WDT_PRIORITY
 
+#define CONFIG_CC310_BASE_ADDR          NORDIC_NRF_CC310_5002A000_BASE_ADDRESS
```

Comment:
This is only applicable to nRF52840, I believe. So could we put this in respective #ifdef guards?
For example, as we do in line 32 for GPIO1

Reply:
It's fixed in newest commit